addons/point_of_sale/models/pos_order.py

```diff
@@ -483,11 +484,12 @@ def _default_pricelist(self):
     company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True, default=lambda self: self.env.user.company_id)
     date_order = fields.Datetime(string='Order Date', readonly=True, index=True, default=fields.Datetime.now)
     user_id = fields.Many2one(
-        comodel_name='res.users', string='Salesperson',
+        comodel_name='res.users', string='User',
         help="Person who uses the cash register. It can be a reliever, a student or an interim employee.",
         default=lambda self: self.env.uid,
         states={'done': [('readonly', True)], 'invoiced': [('readonly', True)]},
     )
+    salesperson = fields.Char(string="Salesperson", compute="_compute_salesperson")
```

Comment:
I removed it. It is now added to pos_order.py in pos_hr and the name was changed to cashier to avoid confusion. We need it because the cashier can be either a user or an employee.